doc/doxygen/references.bib

```diff
@@ -904,146 +871,144 @@ @article{Riviere1999
 
 
 @book{Pelteret2019a,
-  author    = {Pelteret, J-P. V. and Steinmann, P.},
-  publisher = {De Gruyter Mouton},
+  author    = {J.-P. Pelteret and P. Steinmann},
```

Comment:
May you please change my initials back to `J-P.`. That's my preferential spelling.

Reply:
Sorry, my comment might have come across as a bit pointed. I see that the other citations for which I'm an author used `J.-P.`, so I guess I should expand a little: Sometimes the publishers give no option for me (and others like me) to specify our preferred initials, so we get "assigned" our initials. When this happens, I use their given spelling because that's how its recorded. But in the event that the bibtex file that the publishers provide use my full name, I choose my preferred spelling. In each of the three publications with attributions to me (Pelteret2019a, Pelteret2018a and davydov2017hp) fall into the latter category, so perhaps I could use the opportunity to ask you to make them consistent with the `J-P.` spelling. Thanks a lot :-)

Reply:
Of course. Should I then use this convention for all the compound nouns?

Reply:
I must admit that I don't have a good answer for you. I'm in a position of privilege that I get to ask for this change (and I got asked similar in https://github.com/dealii/publication-list/issues/280). In general, I would say that we should stick with what's recorded by the publishers. But that's a large task, and should really be something that's done by whoever added these entries in the first place. So if you feel up to checking this, then I think that its one of those tasks that will be unnoticed by most but could be appreciated by one or two of the authors. Otherwise, I'd say that you should leave things as they are (in the hope that they were correctly added in the first place). You've done a lot of work here, which is already greatly appreciated.

Reply:
I can definitely check all the entries containing compound nouns. However, I will wait for feedback regarding the abbreviation of first and middle names (see the comment by Marc Fehling below) before doing a new commit.